src/main/java/org/cyclops/integrateddynamics/part/aspect/read/AspectReadBuilders.java

```diff
@@ -394,6 +398,15 @@ public ITemperature getOutput(Pair<PartTarget, IAspectProperties> input) {
 
     public static final class Network {
 
+        public static final IAspectPropertyTypeInstance<ValueTypeInteger, ValueTypeInteger.ValueInteger> PROPERTY_CHANNEL =
+                new AspectPropertyTypeInstance<>(ValueTypes.INTEGER, "aspect.aspecttypes.integrateddynamics.integer.channel.name");
+        public static final IAspectProperties PROPERTIES = new AspectProperties(ImmutableList.<IAspectPropertyTypeInstance>of(
+                PROPERTY_CHANNEL
+        ));
+        static {
+            PROPERTIES.setValue(PROPERTY_CHANNEL, ValueTypeInteger.ValueInteger.of(IChanneledNetwork.WILDCARD_CHANNEL));
```

Comment:
If we keep this property, it might be better to set the default channel to channel 0. Because if players want to use a different channel instead, _all_ channels will have to be updated from wildcard to something else, instead of only changing one channel 0 to something else.

Reply:
The rest of the properties here are about the entire network, though, so shouldn't this one also be that way by default?

Reply:
Yes, makes sense. I thought this property had another purpose, you can ignore my comment. (and also the one in IT then)